polygeist/tools/cgeist/Test/Verification/redstore.c

```diff
@@ -1,4 +1,4 @@
-// RUN: cgeist %s --function=* --detect-reduction -S | FileCheck %s
+// RUN: cgeist -O2 %s --function=* -S | FileCheck %s
```

Comment:
could use `-O1` (same for other modified test cases)

Reply:
Well I think -O2 is what the `clang` driver pass down to cgeist by default when we compiler SYCL code so ...

Reply:
Sure.